README.md

```diff
@@ -18,10 +19,9 @@ In order to deploy the tracker, some boilerplate code is neccessary for configur
 1. Clone this repository, typically I put it in a folder called `donations`, which is the path to which this repo will be referred for the remainder of these instructions:
     ```> git clone https://github.com/GamesDoneQuick/donation-tracker-toplevel.git donations```
 1. Make an empty directory under `donations` called `db`. This is where you can keep the working copy of your sqlite database. By default, the settings are such that one called `db/testdb` will be created, but of course you can modify that to suit your needs.
-1. Make a copy of `example_local.py` under `donations`, and call it `local.py`. This is where you will enter any deployment-specific settings for your instance of the website.
-    ```> cp example_local.py local.py```
-    1. (optional) Change the `NAME` field under the `DATABASES` variable to point at a different location if you wish.
-    2. There are some other config variables related to timezone, e-mail, google docs, and giantbomb's API. None of these are neccessary to get started, and mostly can be ignored unless you are interested in that specific feature. Documentation on these fields is lacking, but it shouldn't be too diffficult to figure out how they work if you take a look at `settings.py`.
+1. In `donations`, copy `.env.example` to `.env`. This is where you will enter any deployment-specific settings for your instance of the website.
+    ```> cp .env.example .env```
```

Comment:
Since we're in here, can we remove all the `> ` at the start of the code lines lol.

Makes copying them harder for no real benefit since GitHub stylizes them anyway.